01/userinfamous/wordvalue.py

```diff
@@ -0,0 +1,33 @@
+from data import LETTER_SCORES, DICTIONARY 
+import os
+
+#File path for Dictionary
+fname = os.path.join('../',DICTIONARY)
+
+#Wordlist from dictionary
+def load_words() -> list:
+    with open(fname,'r') as f:
+        wordlist = f.read().split()
```

Comment:
`read()` gives you the whole text in one string and `split()` depends on the default implementation, which is whitespace and line endings. But because the text is already with linebreaks, it is better to use `f.readlines` or to use the `splitlines()` method for strings